mkdocs.yml

```diff
@@ -64,6 +66,10 @@ pages:
   - 'Manage SSL Certificates': 'how-to/manage-certificates.md'
   - 'Manage SQL Scripts': 'how-to/manage-sql-scripts.md'
   - 'Manage Code Repository': 'how-to/manage-code-repository.md'
+  - 'Manage Users': 'how-to/manage-users.md'
+  - 'Manage Authentication Certificates': 'how-to/manage-certs.md'
```

Comment:
perhaps under an auth tree eg auth/manage-certificates.md  and auth/manage-policies, roles, users, etc.